code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -144,6 +147,9 @@
 		/*if(prob(25))
 			location.ReplaceWithSpace()
 			return 0*/
+	if(world.time - sound_start_time >= sound_length) //looping sound
```

Comment:
Use a LOW prob instead

Reply:
the point is to repeat the sound once it finishes, cyberboss
I'm thinking using the ```repeat``` var of sound datums would be better (should be entirely client side cost) but would probably need a dedicated sound channel